AppCenterData/AppCenterData/Internal/Client/MSDataOperationProxy.m

```diff
@@ -199,45 +248,15 @@ - (BOOL)shouldAttemptRemoteOperationForDocument:(MSDocumentWrapper *)document {
 }
 
 /**
- * Update the local store given a current/new cached document.
+ * Returns a flag indicating if a remote operation should be attempted.
+ *
+ * @param partition The partition under which to check for pending operations.
  *
- * @param token The CosmosDB token.
- * @param currentCachedDocument The current cached document.
- * @param newCachedDocument The new document that should be cached.
- * @param deviceTimeToLive The device time to live for the new cached document.
- * @param operation The operation being intended (nil - read, CREATE, UPDATE, DELETE).
+ * @return YES if a remote operation should be attempted; NO otherwise.
  */
-- (void)updateLocalStore:(MSTokenResult *)token
```

Comment:
Why was this removed?

Reply:
Its moved to the Storage class level cos list api would need it extensively